examples/OceananigansSingleColumnSimulation/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
+CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
```

Comment:
Shouldn't this be `ClimaParams`?

Reply:
Yep, this PR started before the name change. I'll update it and merge it today.